ecc/bn254/fp/element_mul_amd64.s

```diff
@@ -313,12 +313,12 @@ TEXT ·mul(SB), $24-24
 	ADCXQ AX, BX
 	ADOXQ BP, BX
 
-	// reduce element(R14,R15,CX,BX) using temp registers (R13,SI,R12,R11)
-	REDUCE(R14,R15,CX,BX,R13,SI,R12,R11)
+	// reduce element(R14,R12,CX,BX) using temp registers (R13,SI,R13,R11)
+	REDUCE(R14,R12,CX,BX,R13,SI,AX,R11)
```

Comment:
I followed up to here.
R15 becomes R12.
R12 becomes R13.
But why AX here?

Reply:
It's indeed a typo - `AX` is used as a temp register by `REDUCE`. Will fix - thx Dave